src/components/Icon.stories.tsx

```diff
@@ -181,7 +183,7 @@ export const Icon = (props: IconProps) => {
       <ul css={{ gap: 24, listStyle: "none", gridTemplateColumns: "repeat(4, 1fr)", ...Css.dg.p0.$ }}>
         {miscIcons.map((icon, i) => (
           <li css={{ gap: 8, ...Css.xsMd.df.aic.fdc.$ }} key={icon}>
-            <IconComponent icon={icon} data-testid={icon} id={icon} color={props.color} />
+            <IconComponent icon={icon} data-testid={icon} id={icon} color={props.color}/>
```

Comment:
**request:** Can you revert this change back to including that space? 
```suggestion
            <IconComponent icon={icon} data-testid={icon} id={icon} color={props.color} />
```

Reply:
@bdow 

Sure, thank you!